documents/TestSuite/sxpbrlib/materials/surfaceshader.mtlx

```diff
@@ -62,7 +62,7 @@
 
   <nodedef name="ND_testshader3" type="surfaceshader" node="testshader3">
     <input name="reflectivity" type="color3" />
-    <input name="edgetint" type="color3" />
+    <input name="edgecolor" type="color3" />
```

Comment:
Changing to "edgecolor" since that is the term used in the white paper now. (the word "tint" is used on parameter that is not physically based and more for artistic control).